.github/workflows/deploy-dev-gcp.yaml

```diff
@@ -34,13 +34,13 @@ jobs:
         run: |
           docker build -f .docker/Dockerfile -t ${IMAGE_GCP} .
       - name: Log in to the container registry
-        uses: docker/login-action@v2.1.0
+        uses: docker/login-action@v3
```

Comment:
Alternativt kan 3 steg ("Build Docker image", "Log in to the container registry" og "Push Docker image") erstattes av ett ("Create Docker image"), ved å bruke action `nais/docker-build-push@v0`, se f.eks. [deploy.yaml linje 48-55](https://github.com/navikt/pensjonskalkulator-backend/blob/main/.github/workflows/deploy.yaml#L48-L55)